### DISCUSSION TITLE
Lab Assignment #1👩‍💻

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/24SU-ASIAAM-191A#26] [Labs]
## Due 7/3 before class 

Please use this thread to post your [lab assignment's #1](https://albertkun.github.io/24SU-ASIAAM-191A/assignments/week1/lab_assignment/) link, which is the GitHub pages URL.

For example:
https://albertkun.github.io/24su-asiaam-191a-sample/week1/index.html

**Comment by giaari15** [comment]
https://giaari15.github.io/24su-aas191-labs/week%201/index.html

**Reply by albertkun** [reply]
hello Gia! is that a dark souls reference? 😄 also, remember to use this for the `style` property in your map variable declaration: 
`    style: 'https://api.maptiler.com/maps/streets/style.json?key=wsyYBQjqRwKnNsZrtci1', // Your style URL`

so it should look like this:
```js
// Initialize the map
const map = new maplibregl.Map({
    container: 'map', // container ID
    style: 'https://api.maptiler.com/maps/streets/style.json?key=wsyYBQjqRwKnNsZrtci1', // Your style URL
    center: [-118.45, 34.05], // Starting position [lng, lat]
    zoom: 12 // Starting zoom level
});
```

this was my bad because I didn't provide that correct code during lab! 😓

**Comment by kianaptran** [comment]
https://kianaptran.github.io/24su-asiaam-191a-labs/week%201/

**Reply by kianaptran** [reply]
https://kianaptran.github.io/24su-asiaam-191a-labs/week%201%20profile/
This is the updated finished lab hehe!

**Comment by stinest** [comment]
https://stinest.github.io/24su-191a-labs/week1/index.html

**Comment by sjeyakum** [comment]
https://sjeyakum.github.io/24su-aas191a-labs/week1/index.html

**Comment by Keiya-Wada** [comment]
https://keiya-wada.github.io/asiaam-labs/week1/index.html

**Comment by kchhoeu** [comment]
https://kchhoeu.github.io/24su-asiaam191a-labs/week1/index.html

**Comment by jasminereyz** [comment]
https://jasminereyz.github.io/AA191-Labs/week1/

**Comment by chrissanrow** [comment]
https://chrissanrow.github.io/asiaam191a-su24/week1/

**Comment by pwinnie2026** [comment]
https://pwinnie2026.github.io/24su-asiaam-191a-labs/Week%201/index.html

**Comment by kimada62** [comment]
https://kimada62.github.io/24su-asiaam-191a-labs/week1more/

**Comment by zhamic7** [comment]
https://zhamic7.github.io/aa191a-labs/week1/index.html

**Comment by nesslee123** [comment]
https://nesslee123.github.io/Assignments/week1/index.html

**Reply by nesslee123** [reply]
There are three markers on my map, but they are far away from each other so you might need to zoom out a little bit.

**Comment by caitlynn-hauw** [comment]
https://caitlynn-hauw.github.io/aa191-assignments/week1/index.html

**Comment by sachigill** [comment]
https://sachigill.github.io/24su-asiaam-191a-labs/week1/

**Comment by NellieNguyen149** [comment]
https://nellienguyen149.github.io/24su-asianam-191a-labs/week1/
